### PR TITLE
add query for stop by id

### PR DESCRIPTION
**Pull request by bas-hbt**

### CODE REVIEW

**Review by jspetrak — APPROVED** [review]

File: src/main/java/org/opentripplanner/client/OtpApiClient.java [hunk 1]
```diff
@@ -113,6 +114,27 @@ public List<Pattern> patterns() throws IOException {
     return deserializeList(json, type, "/data/patterns");
   }
 
+  /**
+   * Returns a TripPlan, also known as a routing result.
+   *
+   * @link <a href="https://docs.opentripplanner.org/api/dev-2.x/graphql-gtfs/queries/plan">OTP API
+   *     docs</a>
+   */
```
Comment (leonardehrenfried) [inline src/main/java/org/opentripplanner/client/OtpApiClient.java:122]: Please update the documentation.

**Comment by leonardehrenfried** [comment]
Thanks for this. If you change the documentation, I can merge this in.

**Review by leonardehrenfried — CHANGES_REQUESTED** [review]
Thanks for this. If you change the documentation, I can merge this in.

File: src/main/java/org/opentripplanner/client/OtpApiClient.java (hunk 1 above)
Comment (bas-hbt) [reply src/main/java/org/opentripplanner/client/OtpApiClient.java]: Classic Copy-Paste-Error. It is fixed now.

**Pull request merged**